albumentations/imgaug/transforms.py

```diff
@@ -22,13 +25,45 @@ def apply(self, img, **params):
 
 
 class DualIAATransform(DualTransform, BasicIAATransform):
-    pass
+    def __init__(self, p):
+        super(DualIAATransform, self).__init__(p)
+
+    def apply_to_bboxes(self, bboxes, **params):
+        rows, cols = params['rows'], params['cols']
```

Comment:
I think we could use `apply_to_bboxes(self, bboxes, rows, cols, **params)` here, since we know that those variables (`rows` and `cols`) should be passed to `apply_to_bboxes`.

Reply:
Totally makes sense.